pkgs/tools/misc/turbo/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub, buildGoModule, protobuf, protoc-gen-go, protoc-gen-go-grpc }:
+let
+  src = fetchFromGitHub {
+    owner = "vercel";
+    repo = "turbo";
+    rev = "69681a3396c377f5bc663bf3f942a62034694f29";
+    sha256 = "csapIeVB0FrLnmtUmLrRe8y54xmK50X30CV476DXEZI=";
+  };
+in
+buildGoModule {
+  pname = "turbo";
+  version = "1.6.3";
+
+  src = "${src}/cli";
+
+  vendorSha256 = "Kx/CLFv23h2TmGe8Jwu+S3QcONfqeHk2fCW1na75c0s=";
+  nativeBuildInputs = [
+    protobuf
+    protoc-gen-go
+    protoc-gen-go-grpc
+  ];
+
+  preBuild = ''
+    make compile-protos
+  '';
+
+  doCheck = false;
```

Comment:
Please add the reason why checks are disabled as comments here.

Reply:
Added comment

Reply:
lets fix that test

https://github.com/vercel/turbo/blob/0fe2584912a8908907a7216a8628871d396f3e4f/cli/internal/hashing/package_deps_hash_test.go#L194

```go
func Test_getTraversePath(t *testing.T) {
	fixturePath := getFixture(1)

	tests := []struct {
		name     string
		rootPath turbopath.AbsoluteSystemPath
		want     turbopath.RelativeUnixPath
		wantErr  bool
	}{
		{
			name:     "From fixture location",
			rootPath: fixturePath,
			want:     turbopath.RelativeUnixPath("../../../"),
			wantErr:  false,
		},
		{
			name:     "Traverse out of git repo",
			rootPath: fixturePath.UntypedJoin("..", "..", "..", ".."),
			want:     "",
			wantErr:  true,
		},
	}
	for _, tt := range tests {
		t.Run(tt.name, func(t *testing.T) {
			got, err := getTraversePath(tt.rootPath)
			if (err != nil) != tt.wantErr {
				t.Errorf("getTraversePath() error = %v, wantErr %v", err, tt.wantErr)
				return
			}
			if !reflect.DeepEqual(got, tt.want) {
				t.Errorf("getTraversePath() = %v, want %v", got, tt.want)
			}
		})
	}
}
```

maybe change workdir for test?